test/e2e/Tests/build-e2e-test.ps1

```diff
@@ -6,20 +6,17 @@
 
 [CmdletBinding()]
 param(
-    [switch]
-    $Clean,
-
     [Switch]
     $SkipStorageEmulator,
 
     [Switch]
-    $StartCosmosDBEmulator,
+    $StartMSSqlContainer,
 
     [Switch]
     $SkipCoreTools,
 
     [Switch]
-    $SkipBuildOnPack
+    $SkipBuild
```

Comment:
I am not quite sure what this SkipBuild is used for? Seems like we never pass this argument,

Reply:
I was using it locally to test the docker logic further down the scriptfile without needing to rebuild these projects every time. I left it in as it may come in handy for those who want to redeploy docker/azurite locally without interfering with the prior build.

Reply:
Done!